### ISSUE TITLE
Add type_of_flash to deployment

### ISSUE THREAD

**Issue by niconoe** [tdwg/camtrap-dp#44]
In GitLab by @peterdesmet on Sep 8, 2020, 18:48

_Note: this was suggested in the 2020-09-08 EUROMAMMAL technical call_

Cameras use different types of flashes to take pictures in the dark, which might have an effect on the animal (i.e. it won't return to that place). Values could be:

```
white flash
infrared flash
black flash
```

This needs further community feedback before we can decide if and how this should be part of the standard.

**Comment by ddachs** [comment]
good point! I would see more categories though: 
-no flash
-white LED flash
-infrared LED flash
-black LED flash
-white Xenon flash

A Xenon white flash is very different from a LED white flash. It is brighter, thus allowing shorter exposure times, which leads to a sharper image. For lynx identification I would not recommend a white LED flash, because the coat markings are not sharp enough.
There is also the possibility of a broken or deactivated flash, or even models without a flash (Timelapse cameras). If we had the information, we could programmatically exclude the dark hours during the night from the deployment.

**Comment by Tim-Hofmeester** [comment]
Good point and great to add. I agree with @ddachs and the list he suggests. We should indeed not forget that there are many camera trappers out there (especially those working with big cats) that still use xenon flash. The type of flash influences not only animal behaviour, but also response time for retriggers of the camera (lower for xenon flash) as well as detection distance (lower for black LED compared to infrared LED for example).
